presto-parser/src/test/java/com/facebook/presto/sql/parser/TestSqlParser.java

```diff
@@ -1746,6 +1746,27 @@ public void testQuantifiedComparison()
                 ));
     }
 
+    @Test
+    public void testQuotedSchemaCreation() throws Exception
+    {
+        String raw = "create schema \"123\"";
+        assertThatFormattedSqlEquals(raw);
```

Comment:
The `assertStatement()` helper already handles this. Add this test to the existing `testCreateSchema()` method. Also, use a name that requires quoting, such as one containing a space:
```java
assertStatement("CREATE SCHEMA \"test abc\"",
        new CreateSchema(QualifiedName.of("test abc"), false, ImmutableMap.of()));
```